lldb/test/API/functionalities/asan/TestMemoryHistory.py

```diff
@@ -36,34 +42,61 @@ def setUp(self):
         self.line_breakpoint = line_number("main.c", "// break line")
 
     # Test line numbers: rdar://126237493
-    def libsanitizer_tests(self):
-        target = self.createTestTarget()
+    # for libsanitizers and remove `skip_line_numbers` parameter
+    def check_traces(self, skip_line_numbers):
```

Comment:
I think you need to say skip_line_numbers=False here, or you'll get TypeError from a call to self.check_traces(), no?

Reply:
Good catch, thanks!
I solved this buy giving the parameter a default value: `check_traces(self, skip_line_numbers=False)`